TableSheet.md

```diff
@@ -0,0 +1,239 @@
+# SilverBullet Markdown Table + Formulajs evaluator
+
+This feature allows you to embed dynamic formulas in your Markdown tables. You can use the `F` function to evaluate formulas and display their results in your tables.
+
+The `F` function takes two arguments:
+- `formulajs`: The formula to evaluate using the Formulajs library.
+- `label` (optional): A label to differentiate between the same formula in different tables.
+
+Here's an example usage: `${F("SUM(A1:A5)","1")}`
+
+```space-lua
+-- ---------------------------
+-- SilverBullet Markdown Table + Formulajs evaluator
+-- Use: F(formulajs,label)
+-- label is optional but necessary if differenciate same formula is different table
+-- Example: ${F("SUM(A1:A5)","1")}
+
+-- Instructions:
+-- not use pcall
+-- not use javascript library
+-- every function must be surrounded by debug_log
+-- not use lua standard methods but space-lua silverbullet api (https://silverbullet.md/API)
+-- string.byte(s, i?, j?)
+-- string.char(...)
+-- string.find(s, pattern, init?, plain?)
+-- string.format(format, ...)
+-- string.gsub(s, pattern, repl, n?)
+-- string.match(s, pattern, init?)
+-- string.gmatch(s, pattern)
+-- string.len(s)
+-- string.lower(s)
+-- string.upper(s)
+-- string.rep(s, n, sep?)
+-- string.reverse(s)
+-- string.sub(s, i, j?)
+-- string.split(s, sep)
+-- string.startsWith(s, prefix)
+-- string.endsWith(s, suffix)
+-- string.trim(s)
+-- string.trimStart(s)
+-- string.trimEnd(s)
+-- string.matchRegex(s, pattern)
+-- string.matchRegexAll(s, pattern)
+-- ---------------------------
+
+LOG_ENABLE = false
+function debug_log(message)
+  if LOG_ENABLE then
+    js.log("[DEBUG] " .. message)
+  end
+end
+
+-- Import Formulajs
+local formulajs = js.import("https://esm.sh/@formulajs/formulajs")
```

Comment:
May be wise to pin tis to a particular version (you can do this by appending `@1.x` or whatever at the end)